pkg/lib/config/authentication.go

```diff
@@ -130,7 +130,7 @@ var _ = Schema.Add("RecoveryCodeConfig", `
 	"type": "object",
 	"additionalProperties": false,
 	"properties": {
-		"count": { "type": "integer" },
+		"count": { "type": "integer", "minimum": 10 },
```

Comment:
Can you also add a reasonable constraint on maximum number? (e.g. ~50)

We would not want developers to spam thousands of recovery code entries per user in database.

Reply:
sure, will add this now